docs/source/05_data/01_data_catalog.md

```diff
@@ -589,3 +590,34 @@ io.save("ranked", ranked)
 ```
 
 > *Note:* Saving `None` to a dataset is not allowed!
+
+# Using Minio (S3 API Compatible Storage) in Data Catalog
```

Comment:
Rather than having a separate section for Minio, could we have it under "Using the Data Catalog with the YAML API" above? You can see there are 13 examples like "Example1, Example2 ...", and I think minio could be "Example 14" :)

Reply:
Updated :)

Reply:
Sorry I reverted and push again, accidentally adding everything in staging from  codespace...